libs/hardwareWallet/ledger/ledgerDeviceListener/subscribeToLedgerDevices.js

```diff
@@ -3,5 +3,5 @@ import { LEDGER_HW_HID_EVENT } from '@libs/hardwareWallet/ledger/constants';
 const IPC = window.ipc;
 
 export const subscribeToLedgerDeviceEvents = (fn) => {
-  IPC.on(LEDGER_HW_HID_EVENT, (action, data) => fn(data));
+  IPC[LEDGER_HW_HID_EVENT]((action, data) => fn(data));
```

Comment:
```suggestion
  IPC[LEDGER_HW_HID_EVENT]((_, data) => fn(data));
```
you can do the same at other places?